prime-router/metadata/fhir_mapping/hl7/resource/Specimen.yml

```diff
@@ -29,6 +29,23 @@ type:
   expressionType: resource
   specs: SPM.4 | SPM.5
 
+extension_1-specimenTypeModifier:
+  condition: $spm5 NOT_NULL && $spm4 NOT_NULL
```

Comment:
I wasn't 100% what 30L is guarding against, but it seemed like a fair check regardless to only map this when both segments are populated